src/exercism/Dog.java

```diff
@@ -0,0 +1,39 @@
+class Dog {
+    String name;
+    String breed;
+
+    // get the name and breed
+    public Dog(String name, String breed) {
+        this.name = name;
+        this.breed = breed;
+    }
+
+    // reset the name and breed
+    public void setNameBreed(String setName, String setBreed) {
+        name = setName;
+        breed = setBreed;
+    }
+
+    // print the name and breed
+    void print() {
+        System.out.println(name + " is a " + breed);
+    }
+
+    // print the resetted name and breed
+    void setPrint() {
```

Comment:
this is incorrect.  The goal is to "modify the attributes using the setter methods".

Each set method, should modify the instance variables.

This set method doesn't modify anything.